conf/storm.yaml.example

```diff
@@ -19,7 +19,7 @@
 #     - "server1"
 #     - "server2"
 # 
-# nimbus.host: "nimbus"
+# nimbus.seeds: ["localhost"]
```

Comment:
since for Storm HA there can be multiple nimbus hosts, shouldn't we provide an example like with storm.zookeeper.servers that includes multiple hosts? Like 

nimbus.seeds: ["host1", "host2", "host3"]